subjects/java/projects/lets-travel/audit/README.md

```diff
@@ -112,17 +116,17 @@
 
 ###### Does the UI support easy navigation for all user roles?
 
-##### Verify if Travel Managers are able to access analytics about their travel listings and subscriber feedback.
-
-###### Do Travel Managers have access to analytics on their listings?
-
 ##### Check if the system adheres to data protection regulations and privacy laws.
 
 ###### Does the system comply with data protection
 
 ##### Verify if development and deployment practices align with industry standards for security and code quality, such as `OWASP` and `clean code principles`.
 
-###### Are development and deployment practices in line with industry standards for security and code quality such as `OWASP` and `clean code principles`?
+###### Is the code readable and simple to understand?
```

Comment:
This is still vague and subjective

Reply:
Okay, do you have any idea how to make it precise ?

Reply:
i removed both term 'clean code principle' and 'OWASP ' and i replace it by asking the auditor to check code readability and separation and also to check about SQL injection and XSS.